python/orca/src/bigdl/orca/data/pandas/preprocessing.py

```diff
@@ -19,10 +19,16 @@
 from bigdl.orca import OrcaContext
 from bigdl.orca.data import SparkXShards
 from bigdl.orca.data.utils import *
-from bigdl.dllib.utils.log4Error import *
+from bigdl.dllib.utils.log4Error import invalidInputError
+from typing import TYPE_CHECKING, List, Optional
 
+if TYPE_CHECKING:
+    from bigdl.orca.data.shard import SparkXShards
+    from pyspark.sql.types import StructType
+    import os
```

Comment:
why need import os?

Reply:
> why need import os?

Actually in preprocessing.py, `import os` is missing. So it is needed and out of this block.